contractcourt/channel_arbitrator.go

```diff
@@ -3105,8 +3109,8 @@ func (c *ChannelArbitrator) channelAttendant(bestHeight int32,
 			// We'll advance our state machine until it reaches a
 			// terminal state.
 			_, _, err = c.advanceState(
-				uint32(bestHeight), breachCloseTrigger,
```

Comment:
actually I think the bestHeight is correct here, because you don't want to log in the `stateStep` actions to the breachClose height but rather the best height.

Reply:
can you elaborate? cause in most cases the breach height should be the current height, suppose the node is always online, then it's the best height. when it's not, it can only be the breach height is in the past, which should be the correct "trigger" height?

Reply:
ahh ok, yeah this trigger height in the breach case seems to not be used other than checking for commit actions which will however have no impact on the state-flow for the breach case. So I think we are good here using any height here.